src/current/v25.4/read-from-standby.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Read from Standby
+summary: Direct read-only queries to your standby cluster instead of your primary cluster.
+toc: true
+docs_area: manage
+---
+
+In addition to providing [failover]({% link {{ page.version.version }}/failover-replication.md %}) capabilities for disaster recovery, CockroachDB **physical cluster replication (PCR)** allows you to direct read-only queries to your standby cluster. This process offloads traffic such as application reads, analytics queries, and ad-hoc reporting from the primary cluster.
+
+Use this page to understand how the **read from standby** feature works and how to utilize it.
+
+## How the read from standby feature works
+
+PCR utilizes cluster virtualization to separate clusters' control planes from their data planes. A cluster always has one control plane, called a _system virtual cluster (SystemVC)_, and at least one data plane, called an _App Virtual Cluster (AppVC)_. A cluster's SystemVC manages PCR jobs and cluster metadata, and is not used for application queries. All data tables, system tables, and cluster settings in the standby cluster's AppVC are identical to the primary cluster's AppVC. The standby cluster's AppVC itself remains offline during replication.
+
+When using read from standby, applications can read from the standby cluster, but they do not connect directly to the standby cluster's AppVC. Instead, PCR introduces a _reader virtual cluster (ReaderVC)_. The ReaderVC ensures a clean, isolated environment specifically for serving read queries without interfering with replication or system metadata. It reads continuously from the standby cluster's AppVC using internal pointers, providing access to the replicated data while keeping the AppVC offline. The ReaderVC does not store any data itself, so it does not require extra disk space.
```

Comment:
>The ReaderVC does not store any data itself, so it does not require extra disk space

This isn't quite true. I would say "the readerVC does not store any user data, only up to a couple GB of metadata."